### ISSUE TITLE
Bug in Exo Bundle, open question (distribution bundle) - answers

### ISSUE THREAD

**Issue by CPASimUSante** [claroline/Distribution#326]
**problem 1:**
Following the test in #324 , i gave a non empty string for the answers, and when i clicked on "finish exercise" the answers are processes. But,
1 - There are console.log data thrown in the JS console (ok in dev... but could be harmful in prod for IE browser)
2 - there is a js error thrown : 
```
TypeError: Cannot read property 'length' of undefined
    at AbstractQuestionCtrl.onFeedbackShow (ujm-exo-app.js:134)
    at FeedbackService.executeCallbacks (ujm-exo-app.js:32)
    at FeedbackService.showFeedback [as show] (ujm-exo-app.js:32)
    at QuestionShowCtrl.<anonymous> (ujm-exo-app.js:45)
    at ujm-exo-angular-js.js:9
    at e (ujm-exo-angular-js.js:4)
    at ujm-exo-angular-js.js:4
```
=> the field "Your answer" is empty

If i refresh the page, the user answers are displayed and the error does not appear again.

THEN **problem2**
on that display answer page, if i click on the top button "Answer" again (to have a new try at the exercise) i land on the page where i can answer, BUT the bottom buttons are not the same : 
My exercise is composed with 1 step and 2 questions.
The first time i answer i have the button : 
"Finish the exercise" (previous and next button are greyed out)
The second time i have the green button "valid the step" (only clickable), and "Finish the exercise" (that is not clickable)
If i answer and click on valid the step, nothing happens... I have to get out the exercise, come back and i have the "Finish the exercise" button working again (and no green "valid step" button)

**Comment by Elorfin** [comment]
OK, I think the second problem is the consequence of the first one (by the way I hope).
The green "valid step" button is only for Formative exercise (there is a problem with display conditions).

You talk about IE, is that specific to this browser or do you have the problem in FF or Chrome ?

In any case, I will remove all the forgotten `console.log`.

**Comment by Elorfin** [comment]
In fact the two problems are not related.
I have opened an issue for the second one : #335.

**Comment by Elorfin** [comment]
the problem 1 will be fixed by the merge of #347.
the problem 2 is already resolved.
